docs/_snippets/patient.mdx

```diff
@@ -42,16 +42,32 @@
       The ID number.
     </ParamField>
 
+    <ParamField body="assigner" type="string" optional>
+      The assigning entity
```

Comment:
let's add an example here, I don't know what this means - ie is it a name of a Driver's licensing office?